content/en/blog/_posts/2023-08-21-job-update-post.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: blog
+title: "Kubernetes 1.28: New Job Features"
+date: 2023-08-21
+slug: kubernetes-1-28-jobapi-update
+---
+
+**Authors:** Kevin Hannon (G-Research), Michał Woźniak (Google)
+
+This blog discusses two new features in Kubernetes 1.28 to improve Jobs for batch
+users: [PodReplacementPolicy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3939-allow-replacement-when-fully-terminated)
+and [BackoffLimitPerIndex](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs).
+
+## Pod Replacement Policy
+
+### What problem does this solve?
+
+By default, when a pod enters a terminating state (e.g. due to preemption or
+eviction), a replacement pod is created immediately, and both pods are running
+at the same time.
+
+This is problematic for some popular machine learning frameworks, such as
+TensorFlow and [JAX](https://jax.readthedocs.io/en/latest/), which require at most one pod running at the same time,
+for a given index (see more details in the [issue](https://github.com/kubernetes/kubernetes/issues/115844)).
```

Comment:
Are there that many more details there? I think it's ok to skip, as issues are generally very disorganized contents.

Reply:
Changes are still feasible if we want to make them.